static/sass/styles.scss

```diff
@@ -294,9 +294,35 @@ html {
   display: inline-block;
 }
 
-.middle-page-tabs {
-  @media screen and (min-width: $breakpoint-large) {
-    padding-left: 20.3rem;
+.p-tabs--25-75 {
+  @extend %vf-row;
+  position: relative;
+  margin-bottom: 1.5rem;
```

Comment:
I'd suggest using the variables `$spv--x-large` (same as Vanilla is using for this margin): 

https://github.com/canonical/vanilla-framework/blob/cc34bd307a80189d3bc00a851f0ed59c4c6be9a2/scss/_patterns_tabs.scss#L14